src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  @include circle($stopwatch-size);
+  margin: 0 auto;
+  border: 5px solid $stopwatch-color;
+  cursor: pointer;
+
+  &:hover .animated {
+    animation-play-state: paused;
+  }
+
+  &__seconds {
+    position: absolute;
+    top: 8%;
+    z-index: 1;
+    width: $arrow-width;
+    height: 250px;
+
+    background-color: $stopwatch-color;
+    border-radius: $arrow-border-radius;
+
+    transform-origin: center bottom;
+    animation: reversal
+      60s
+      $arrow-timing-f
+      60
+      $arrow-direction
+      $arrow-fill-mode;
+  }
+
+  &__minutes {
+    position: absolute;
+    top: 16%;
+    width: $arrow-width;
+    height: 200px;
+    background-color: darkblue;
+    border-radius: $arrow-border-radius;
+
+    transform-origin: center bottom;
+    animation: reversal
+      3600s
+      $arrow-timing-f
+      1
+      $arrow-direction
+      $arrow-fill-mode;
+  }
+
+  &__dot {
+    position: absolute;
+    top: 47.5%;
```

Comment:
don't use magic numbers, calculate position basing on dot size

Reply:
Ok, I will try to change it today